### ISSUE TITLE
 az artifacts universal download fails

### ISSUE THREAD

**Issue by jishiping** [Azure/azure-cli-extensions#780]
## Describe the bug

**Command Name**
`az artifacts universal download
Extension Name: azure-devops. Version: 0.9.0.`

**Errors:**
```
expected str, bytes or os.PathLike object, not NoneType
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 560, in execute
    raise ex
cli/core/commands/__init__.py, ln 618, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 609, in _run_job
    cmd_copy.exception_handler(ex)
...
az/lib/python3.6/subprocess.py, ln 709, in __init__
    restore_signals, start_new_session)
az/lib/python3.6/subprocess.py, ln 1275, in _execute_child
    restore_signals, start_new_session, preexec_fn)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az artifacts universal download --organization {} --feed {} --name {} --version {} --path {}`

## Environment Summary
```
Linux-4.18.0-20-generic-x86_64-with-debian-buster-sid
Python 3.6.5
Shell: bash

azure-cli 2.0.66

Extensions:
azure-devops 0.9.0

```

**Comment by atbagga** [comment]
@jishiping Can you update the extension and try again? There is a bug fix which went in Azure-devops 0.10.0. 
 
```
az extension update -n azure-devops
```

**Comment by jishiping** [comment]
Thanks @atbagga, download works after extension update!
